client/src/components/Menu/Menu.css

```diff
@@ -0,0 +1,10 @@
+.hamburger {
+	background-color: white;
+}
+
+.burgers {
+    width: 35px;
+    height: 5px;
+    background-color: black;
+    margin: 6px 0;
+  }
```

Comment:
Can you think of more semantic class names here? Something that's more differentiated?